fuzz/Makefile

```
@@ -0,0 +1,10 @@
+.PHONY: fuzz
+fuzz:
+	cargo +nightly fuzz run differential -- -runs=3000
+
+HASH=?
+
+.PHONY: extract-wasm
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)